### PR TITLE
Part 1

### PR DESCRIPTION
**Pull request by VitaliKarpuk**
Part 1

### CODE REVIEW

File: config/webpack/webpack.dev.config.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            loader: "html-loader",
+          }
+        ]
+      },
+      { 
+        test: /\.(s[ca]ss)/,
+        use: [ 'style-loader', 'css-loader', 'sass-loader' ]
+      },
+      // {
+      //   test: /\.(s[ca]ss)/,
```
Comment (omelenetskiy) [inline config/webpack/webpack.dev.config.js]: plz remove extra comments

File: config/webpack/webpack.dev.config.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  },
+  plugins: [
+    new HtmlWebPackPlugin({
+      template: "./src/index.html",
+      filename: "./index.html",
+      excludeChunks: ['server']
+    }),
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoEmitOnErrorsPlugin(),
+    // new MiniCssExtractPlugin({
```
Comment (omelenetskiy) [inline config/webpack/webpack.dev.config.js]: needed?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    "eslint-plugin-promise": "^4.2.1",
+    "express": "^4.16.4",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-test-renderer": "^17.0.1"
+  },
+  "engines": {
+    "node": ">=4.3.2"
+  },
+  "jest": {
```
Comment (omelenetskiy) [inline package.json]: lets move it to separate file jest.config.js

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import MovieDetailsPage  from './pages/MovieDetailsPage/MovieDetailsPage';
+
+const App = () => {
+  console.log('ffff');
```
Comment (omelenetskiy) [inline src/App.js]: useless console log here

File: src/adder.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+const adder = (x, y) => x + y
```
Comment (omelenetskiy) [inline src/adder.js]: I think it's not needed at all

File: src/server/server-dev.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+/* eslint-disable no-undef */
+import path from 'path'
+import express from 'express'
+import webpack from 'webpack'
+import webpackDevMiddleware from 'webpack-dev-middleware'
+import webpackHotMiddleware from 'webpack-hot-middleware'
+import config from '../../config/webpack/webpack.dev.config';
+
+const app = express(),
+  DIST_DIR = __dirname,
```
Comment (omelenetskiy) [inline src/server/server-dev.js]: it is not a good approach to set constants in this way
it works but I would recommend to go with such way:

```
const a = 
const b = 
const c =   
```

File: src/server/server-prod.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
```
Comment (omelenetskiy) [inline src/server/server-prod.js]: probably we don't need two files for server
we can move this logic to the server-dev.js

and then use condition to choose needed part of code

```
 if(production) {
...
} else {
...dev code here
}
```

**Review by omelenetskiy — APPROVED** [review]

File: config/webpack/webpack.dev.config.js (hunk 1 above)
Comment (VitaliKarpuk) [reply config/webpack/webpack.dev.config.js]: done

File: config/webpack/webpack.dev.config.js (hunk 2 above)
Comment (VitaliKarpuk) [reply config/webpack/webpack.dev.config.js]: done

File: package.json (hunk 3 above)
Comment (VitaliKarpuk) [reply package.json]: done

File: src/adder.js (hunk 5 above)
Comment (VitaliKarpuk) [reply src/adder.js]: deleted

File: src/App.js (hunk 4 above)
Comment (VitaliKarpuk) [reply src/App.js]: deleted

File: src/server/server-dev.js (hunk 6 above)
Comment (VitaliKarpuk) [reply src/server/server-dev.js]: done

File: src/server/server-prod.js (hunk 7 above)
Comment (VitaliKarpuk) [reply src/server/server-prod.js]: done

**Pull request merged**